packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -279,7 +276,7 @@ class DDSFooterComposite extends ModalRenderMixin(HybridRenderMixin(HostListener
                 value="${selectedLanguage}"
                 @bx-combo-box-selected="${languageCallback}"
               >
-                ${parsedList?.map(
+                ${langList?.map(
```

Comment:
Given `language` is the only one of our interest in `langList`, can we make `langList` a string array?